content/guides/user-guides/multi-cloud/object-archival-restore/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archival and Retrieval
+description: "User guide for Object Archival and Retrieval"
+weight: 40
+disableToc: false
+tags: ["user guide", "archival and retrieval"] 
+---
+
+## Introduction to Object archival and retrieval
+SODA Multicloud provides a cloud vendor agnostic data management for hybrid cloud, intercloud, or intracloud. The goal is to provide a unified interface to support file, block, and object services across multiple cloud vendors.
+
+Archiving data which is important but intermittently used, is valuable to keep it safe and also save cost and resource.
+Retrieving the archived data whenever required as per need and also retrieving it when the local data is lost is an important requirement for Availability and Business continuity.
+More details about the architecture and use cases can be found at [ObjectArchivalRestore](https://github.com/sodafoundation/architecture-analysis/tree/master/arch-design/multicloud)
+
+## Getting Started
+This guide shows how to archive objects into Cloud object storage and restore them as and when required, meeting the SLAs. This will illustrate the operations through SODA Dashboard.
+
+To refer the API specs, please check [Cloud Object Archival API Specs](https://github.com/sodafoundation/multi-cloud/blob/master/openapi-spec/swagger.yaml)
+
+> \[*NOTE*\]: Currently this feature is available only for AWS cloud
```

Comment:
nit: use the [notice shortcode](https://learn.netlify.app/en/shortcodes/notice/) to highlight notes

Reply:
Done